librubyfmt/src/string_escape.rs

```diff
@@ -17,7 +17,7 @@ pub fn single_to_double_quoted(content: String, start_delim: &str, end_delim: &s
         };
 
         let regexp = Regex::new(&format!(
-            r#"(?<!\\)(\\\\)*(\"|\\\\{}|\\\\{})"#,
+            r#"(?<!\\)(\\\\)*(\"|\\{}|\\{})"#,
```

Comment:
Not sure why I wrote it with these extra escapes originally, the updated version is the same as [what we used in Ripper](https://github.com/fables-tales/rubyfmt/blob/56cec999d663d1adc665dbec2ace2c8f152dae8b/librubyfmt/rubyfmt_lib.rb#L567-L568).